tests/extensions/default/analyzers/Microsoft.DotNet.UpgradeAssistant.Extensions.Default.Analyzers.Test/TestHelper.cs

```diff
@@ -81,27 +91,33 @@ private static async Task<IEnumerable<Diagnostic>> GetDiagnosticsFromProjectAsyn
                 .Where(d => diagnosticIds.Contains(d.Id, StringComparer.Ordinal));
         }
 
-        public static async Task<Document?> GetSourceAsync(string documentPath)
+        public static async Task<Document?> GetSourceAsync(Language lang, string documentPath)
         {
             if (documentPath is null)
             {
                 throw new ArgumentNullException(nameof(documentPath));
             }
 
             using var workspace = MSBuildWorkspace.Create();
-            var project = await workspace.OpenProjectAsync(TestProjectPath).ConfigureAwait(false);
+            var projectLanguage = lang.GetFileExtension();
+            var path = TestProjectPath.Replace("{lang}", $"Fixed.{projectLanguage}", StringComparison.OrdinalIgnoreCase);
```

Comment:
Is this supposed to be including 'Fixed.'? It looks like previously, this would just give the path specified without interjecting 'fixed'.

Reply:
Yes, I found that there was a behavioral difference in the testing (when adding VB support) surfacing from 2 classes having exactly the same name, included in the same project file, but not being partials.

You'll see now that we have 2 csproj files and 2 vbproj files to manage the before and after projects.